### ISSUE TITLE
How to continue to train?

### ISSUE THREAD

**Issue by GulerEnes** [Akhilesh64/ResUnet-a#4]
Hi, I am trying to use your codes on Google Colab. But sometimes Colab suddenly closes itself. Is there a way to continue training?

I didn't see any flag in main.py about this.

**Comment by Akhilesh64** [comment]
Hi @GulerEnes this is an issue with colab. It shuts down if no activity is detected for sometime. There is no problem with the code.

**Comment by GulerEnes** [comment]
> Hi @GulerEnes this is an issue with colab. It shuts down if no activity is detected for sometime. There is no problem with the code.

Yes, I know the codes are working well. Each 10 epoch a new model.h5 file saving. I just want to continue the training from the last saved model.h5 for any reason. How can I do this?

**Comment by Akhilesh64** [comment]
The scripts don't yet support resuming from checkpoint but I'll try to implement this if I get time or you can do these changes yourselves because this seems pretty straightforward.
